src/frontend/templates/header.html

```diff
@@ -46,6 +46,13 @@
 
 <body>
     <header>
+        {{ if $.frontendMessage }}
+        <div class="navbar">
+            <div class="container d-flex justify-content-center">
+                <div class="h-free-shipping">{{ $.frontendMessage }}</div>
+            </div>
+        </div>
+        {{ end }}
```

Comment:
A note about `{{ if $.frontendMessage }}`: we `$.frontendMessage` will evaluate to false if the string is empty, [according to the docs](https://pkg.go.dev/text/template#hdr-Actions) (and my manual testing):
```
{{if pipeline}} T1 {{end}}
	If the value of the pipeline is empty, no output is generated;
	otherwise, T1 is executed. The empty values are false, 0, any
	nil pointer or interface value, and any array, slice, map, or
	string of length zero.
	Dot is unaffected.
```